plugins/headless/hu.bme.mit.gamma.headless.api/docs/headless-gamma-document.md

```diff
@@ -112,30 +110,40 @@ This error occurs when the "Generate p2 repository" option remains checked when
+
+This error can occur after exporting the Headless Gamma. Along with the exported products, a "logs.zip" is created as well. Inside, one or more (depending on the exported product) folders can be found named after exported plugins. The logs found inside these folders contain text similar to this:
+
+" 5/10/21, 2:32:41 PM CEST  
+Eclipse Compiler for Java(TM) v20210223-0522, 3.25.0, Copyright IBM Corp 2000, 2020. All rights reserved.  
+Compliance level '11' is incompatible with target level '15'. A compliance level '15' or better is required" 
+
+This means that the compiler compliance level is set too high. Open the Eclipse IDE, select Window -> Preferences -> Java -> Compiler, and under "JDK Compliance", set the "Compiler compliance level" to 11. After this, export the products again, and the problem should resolve.
 
 ## Setting up the Docker container
 
  Make sure that Docker is installed on the computer in use. The following tutorial was used to install Docker on Ubuntu: https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-18-04
 
  1. Create a working folder, and copy the Dockerfile inside.
- 2. Inside the working folder, create the following folders: "eclipse", "generator", "server", "uppaal", "theta".
+ 2. Inside the working folder, create the following folders: "eclipse",  "server", "uppaal", "theta".
  3. Export the Headless Gamma API inside the "eclipse" folder.
- 4. Export the Headless Generator inside the "generator" folder.
  5. Copy the "OpenApiGammaWrapper" folder, which contains the webserver, inside the "server" folder.
  6. Copy the contents of the UPPAAL folder installed on your computer inside the "uppaal" folder.
  7. Copy the `get-theta.sh` file inside the "theta" folder. This file can be found inside the `gamma/plugins/xsts/theta-bin` folder. 
  8. Open a terminal and build the docker image with the following command: `docker build -t gamma <path to working folder>` (the working folder is the folder created in the first step) . This will create a docker image named "gamma".
  9. In a terminal, run a docker container from the gamma image using the `docker run -it -p 8080:8080 --network host --name gamma_container gamma:latest` command. This command will bind the localhost:8080 address of the host machine to the Docker container, forwarding commands to the webserver running inside. The container is named "gamma_container", and starts in interactive mode, allowing for CLI access. The last parameter is the image, which is `gamma:latest`, which indicates that the latest Gamma image build is used.
  
 It is possible to change the port binding if the user wishes, but the webserver listens to port 8080. In the port binding parameter, the part before the ":" stands for the host port, and the one after is the container port. So changing the parameter to "5555:8080" is valid, but "8080:5555" would result in communication failure.  Names of the image and container can also be changed.
+
```

Comment:
Delete unnecessary new lines, only one should remain.

Reply:
Deleted unnecessary new lines.